dev/ContentDialog/ContentDialog_themeresources.xaml

```diff
@@ -267,61 +200,112 @@
                                 </VisualState>
                             </VisualStateGroup>
                         </VisualStateManager.VisualStateGroups>
-                        <Grid x:Name="LayoutRoot"
-                              contract5Present:Visibility="Collapsed"
-                              Background="{ThemeResource SystemControlPageBackgroundMediumAltMediumBrush}">
-                            <contract5NotPresent:Grid.ColumnDefinitions>
-                                <ColumnDefinition Width="Auto" />
-                            </contract5NotPresent:Grid.ColumnDefinitions>
+                        <Grid x:Name="LayoutRoot">
+                            <Rectangle x:Name="SmokeLayerBackground" Fill="{ThemeResource ContentDialogSmokeFill}"/>
                             <Border
                                 x:Name="BackgroundElement"
                                 Background="{TemplateBinding Background}"
                                 FlowDirection="{TemplateBinding FlowDirection}"
                                 BorderThickness="{TemplateBinding BorderThickness}"
                                 BorderBrush="{TemplateBinding BorderBrush}"
-                                contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
-                                contract7NotPresent:CornerRadius="{ThemeResource OverlayCornerRadius}"
-                                MinWidth="{ThemeResource ContentDialogMinWidth}"
-                                MaxWidth="{ThemeResource ContentDialogMaxWidth}"
-                                MinHeight="{ThemeResource ContentDialogMinHeight}"
-                                MaxHeight="{ThemeResource ContentDialogMaxHeight}"
+                                CornerRadius="{ThemeResource OverlayCornerRadius}"
```

Comment:
Should use CornerRadius="{TemplateBinding CornerRadius}" to allow devs to more easily customize this value if they want without having to include entire template (also backward compatible). Add <Setter Property="CornerRadius" Value="{ThemeResource OverlayCornerRadius}" /> above. Also, removing contract7Present makes me nervous. As far as I'm aware we still need this for platforms that don't support CornerRadius. Am I wrong?

Reply:
Yes, WinUI supports platforms that don't support CornerRadius so removing the contract check sounds like a possible crash on lower platforms.

Reply:
The thing that is weird is above both contract7Present and contract7NotPresent apply to CornerRadius which seems wrong and a bug:
`contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
                                contract7NotPresent:CornerRadius="{ThemeResource OverlayCornerRadius}"`

Reply:
Updated

Reply:
@marksfoster It might be that certain resources only exist in some OS version. IIRC OverlayCornerRadius was introduced later and wasn't available from the start.